repos/fdbt-site/src/utils/apiUtils/matching.ts

```diff
@@ -38,7 +40,7 @@ export const getFareZones = (
 
 export const getMatchingFareZonesAndUnassignedStopsFromForm = (
     req: NextApiRequest,
-): { matchingFareZones: MatchingFareZones; unassignedStops: Stop[] } => {
+): { matchingFareZones: MatchingFareZones; unassignedStops: UnassignedStop[] } => {
     const matchingFareZones: MatchingFareZones = {};
     const bodyValues: string[] = Object.values(req.body);
     const unassignedStops: Stop[] = [];
```

Comment:
Should this be an array of UnassignedStops instead of an array of Stops?

Reply:
hmm ... I don't think I made that change, it doesn't ring a bell for me.